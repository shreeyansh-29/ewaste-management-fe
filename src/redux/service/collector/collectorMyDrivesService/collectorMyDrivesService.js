import {COLLECTOR_DRIVE_MYDRIVE} from "../../../../container/constant/constant";
import api from "../../../../core/utilities/httpProvider";

export const collectorMyDrivesService = () => {
  return api.get(COLLECTOR_DRIVE_MYDRIVE);
};
