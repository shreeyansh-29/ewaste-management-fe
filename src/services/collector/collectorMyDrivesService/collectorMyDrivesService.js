import {COLLECTOR_DRIVE_MYDRIVE} from "../../../container/constant/constants";
import api from "../../../core/utilities/httpProvider";

export const collectorMyDrivesService = () => {
  return api.get(COLLECTOR_DRIVE_MYDRIVE);
};
