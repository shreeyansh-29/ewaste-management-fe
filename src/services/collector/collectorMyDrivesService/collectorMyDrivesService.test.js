/* eslint-disable no-undef */

import React from "react";
import Enzyme, {shallow} from "enzyme";
import {collectorMyDrivesService} from "./collectorMyDrivesService";
import toJson from "enzyme-to-json";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({adapter: new Adapter()});

it("testing collectorMyDrivesService", () => {
  const Service = collectorMyDrivesService;
  const wrapper = shallow(<Service />);
  expect.assertions(1);
  expect(toJson(wrapper)).toMatchSnapshot();
});
