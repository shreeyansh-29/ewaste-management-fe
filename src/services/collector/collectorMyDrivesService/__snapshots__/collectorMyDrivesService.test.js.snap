// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing collectorMyDrivesService 1`] = `<Component />`;
